README.md

```diff
@@ -18,6 +18,15 @@ PRISM Break follows [the GNU/FSF definition of Free Software](https://www.gnu.or
 
 PRISM Break strives to promote the best open source applications. Ease of use, stability, and performance matter. This is the first time many people are looking to leave their proprietary walled gardens. Let's make it a good experience for them. If you're writing a privacy-minded FOSS app, please finish it before asking PRISM Break to promote it.
 
+ - **The software must actually be secure.**
+
+This can vary case-to-case, but generally here are the attributes we look for:
+
+ ** Understand security and take it seriously (this is very broad, but can range from "don't invent your own crypto" to "think about and design against metadata leaks")
+ ** Offers security advantages that other projects don't
+ ** Can keep up with security events
```

Comment:
This is the best point and also relatively easy to evaluate. It should be at the top of the list.

Reply:
Done